src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -36,11 +36,12 @@
 */
 @isTest
 private class ACCT_IndividualAccounts_TEST {
+
```

Comment:
new methods added to this class are 
1) setAdminAccRecTypeAsBizOrgHelper()
2) testContactInsertWithBizAccRecTypeAsAdminAccRecType
3) testContactLastNameUpdateWithBizAccRecTypeAsAdminAccRecType

The other changes to this file are updates to follow EDA styles guide.